applesingle.c

```diff
@@ -1278,7 +1278,7 @@ int main(int argc, char * argv[])
 "Encoding of files is not supported on this platform.\n"
 "\n"
 #endif
-, me, me, me, me, me);
+, me, me, me);
 		rc = 3;
```

Comment:
This patch combines unrelated changes. I think it should be split up.

Reply:
How would you prefer it to be split up? Pretty much the whole PR is changes to printf statements to correct compiler warnings. TBH it was pretty much all just to get it to work on my machine and contribute the changes back in case they are helpful to someone else so i wasnt particularly thinking of trying to build it with different flags or for other architectures